faunadb/client.py

```diff
@@ -138,9 +138,10 @@ def __init__(
         "Accept-Encoding": "gzip",
         "Content-Type": "application/json;charset=utf-8",
         "X-Fauna-Driver": "python",
-        "X-FaunaDB-API-Version": API_VERSION,
-        "X-Query-Timeout": str(self._query_timeout_ms)
+        "X-FaunaDB-API-Version": API_VERSION
       })
+      if self._query_timeout_ms is not None:
```

Comment:
doesn't python handle directly `if self._query_timeout_ms:` instead of the double negation ?

Reply:
Using `is not None` aligns more with the Python style guide

https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
nice.. thanks for the clarification \m/